pinot-common/src/main/java/org/apache/pinot/common/failuredetector/ConnectionFailureDetector.java

```diff
@@ -16,29 +16,18 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.broker.failuredetector;
+package org.apache.pinot.common.failuredetector;
 
 import javax.annotation.concurrent.ThreadSafe;
-import org.apache.pinot.core.transport.QueryResponse;
-import org.apache.pinot.core.transport.ServerRoutingInstance;
 
 
 /**
  * The {@code ConnectionFailureDetector} marks failed server (connection failure) from query response as unhealthy, and
  * retries the unhealthy servers with exponential increasing delays.
+ * <p>
+ * This class doesn't currently implement any additional logic over BaseExponentialBackoffRetryFailureDetector and is
+ * retained for backward compatibility.
```

Comment:
On second thought we could probably remove this class altogether since the class name is not exposed via the user configuration directly.